### DISCUSSION TITLE
A bit confused about the product

### DISCUSSION THREAD

**Discussion by faisalsayed10** [Heroic-Games-Launcher/HeroicGamesLauncher#619] [General]
hey, so i came across this and i installed it in my mac to just check it out. 
So normally, in the epic games store, you cannot install windows games like Watch Dogs / GTA V etc.
But Heroic literally starts the installation of these games without any problem on my m1 mac. 
I also saw that heroic uses wine. does this mean i can play them on my mac too?

**Comment by flavioislima** [comment]
Yes if the game is compatible with wine or crossover.

**Reply by faisalsayed10** [reply]
woah!

**Reply by ghost** [reply]
yup
